### PR TITLE
add Internationalization

### PR DESCRIPTION
**Pull request by YgRamos9**
## Summary

- add Internationalization 
- modify unit tests

## Screenshots

Captura de Pantalla 2022-08-22 a la(s) 3 16 03 p m

## Known Issues

- when entering the translations the pipe translate does not work and the ngx-traslate service could not be loaded

## Trello Card

https://trello.com/c/sP8JRnuX/10-internationalization

### CODE REVIEW

File: angular/src/app/components/language/language.component.ts [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+  ];
+
+  constructor(private translate: TranslateService) {}
+
+  ngOnInit(): void {
+    this.translate.setDefaultLang('es');
+    this.changeLanguage();
+  }
+
+  changeLanguage() {
```
Comment (miguelhenaoa) [inline angular/src/app/components/language/language.component.ts]:
```suggestion
  changeLanguage(): void {
```

File: angular/src/app/screens/auth/components/shopping-cart/shopping-cart.component.html [hunk 2]
```diff
@@ -31,15 +31,17 @@
...
             class="popup-empty-img"
             src="assets/cart-empty.png"
           />
-          <span class="popup-empty-title">El carrito está vacío</span>
-          <span class="popup-empty-desciption">
-            Parece que no tienés items en tu carrito de compras.
+          <span class="popup-empty-title">{{
+            'shopping.emptyTitle' | translate
+          }}</span>
+          <span class="popup-empty-desciption"
```
Comment (miguelhenaoa) [inline angular/src/app/screens/auth/components/shopping-cart/shopping-cart.component.html]:
```suggestion
          <span class="popup-empty-description"
```

File: angular/src/app/screens/auth/components/shopping-cart/shopping-cart.component.scss [hunk 3]
```diff
@@ -98,14 +98,13 @@
   height: 39px;
   line-height: 38px;
   margin: 0 0 24px;
-  text-align: center;
+  display: block;
```
Comment (miguelhenaoa) [inline angular/src/app/screens/auth/components/shopping-cart/shopping-cart.component.scss]: sort

File: angular/src/assets/i18n/es.json [hunk 4]
```diff
@@ -0,0 +1,19 @@
+{
+  "navbar": {
+    "logout": "Cerrar Sesión"
+  },
+
+  "shopping": {
+    "title": "Tu carrito de compras",
+    "emptyTitle": "El carrito esta vacio",
```
Comment (miguelhenaoa) [inline angular/src/assets/i18n/es.json]:
```suggestion
    "emptyTitle": "El carrito está vacío",
```

File: angular/src/assets/i18n/es.json [hunk 5]
```diff
@@ -0,0 +1,19 @@
...
+    "emptyDescription": "Parece que no tienés items en tu carrito de compras."
```
Comment (miguelhenaoa) [inline angular/src/assets/i18n/es.json]:
```suggestion
    "emptyDescription": "Parece que no tienes items en tu carrito de compras."
```
o
```suggestion
    "emptyDescription": "Parece que no tenés items en tu carrito de compras."
```

**Review by joacomiguens — APPROVED** [review]
Amazing Job brooooo 👨🏻‍🚀 I'm so glad of your Angular level. Hope you have great opportunities on EcoPetrol project. Best wishes, SUS 💯

**Review by miguelhenaoa — APPROVED** [review]
Spectacular growth bro, a pleasure to have worked with you.

**Pull request merged**